src/coreclr/jit/compiler.cpp

```diff
@@ -5071,6 +5071,16 @@ void Compiler::compCompile(void** methodCodePtr, uint32_t* methodCodeSize, JitFl
     compJitStats();
 
 #if defined(TARGET_WASM)
+#ifdef DEBUG
+    static ConfigMethodRange EnableLlvmRange;
+    EnableLlvmRange.EnsureInit(JitConfig.EnableLlvmRange());
+
+    if (!EnableLlvmRange.Contains(info.compMethodHash()))
+    {
+        fatal(CORJIT_SKIPPED);
+    }
```

Comment:
This allows one to binary search for methods causing silent bad codegen. The syntax is `--codegenopt:EnableLlvmRange=00000000-FFFFFFFF` (method hashes are unsigned 4-byte integers).